src/scene/materials/standard-material.js

```diff
@@ -26,8 +26,8 @@ const _props = {};
 // special uniform functions on a standard material
 const _uniforms = {};
 
-// temporary helper array
-const _propsSet = [];
+// temporary set of params
+let _params = { };
```

Comment:
personally I'd use a `new Set()` as that would not require to store `true` as a value as currently nor use `hasOwnProperty`

Reply:
Agreed. And you can use `Set#clear` to empty it instead of creating new objects.

Reply:
@mvaligursky Since @slimbuck is away for a while, do you want to have a stab at making this change? Then we can merge...